lua/entities/gmod_wire_expression2/core/hologram.lua

```diff
@@ -359,20 +361,22 @@ local function enable_clip(Holo, idx, enabled)
 	end
 end
 
-local function set_clip(Holo, idx, origin, normal, isglobal)
+local function set_clip(Holo, idx, origin, normal, isglobal, localentid)
 	local clip = check_clip(Holo, idx)
 
-	if clip and (clip.origin ~= origin or clip.normal ~= normal or clip.isglobal ~= isglobal) then
+	if clip and (clip.origin ~= origin or clip.normal ~= normal or clip.isglobal ~= isglobal or clip.localentid ~= localentid) then
 		clip.origin = origin
 		clip.normal = normal
 		clip.isglobal = isglobal
+		clip.localentid = localentid
 
 		add_queue( clip_queue, Holo.e2owner, { Holo,
 			{
 				index = idx,
 				origin = origin,
 				normal = normal,
-				isglobal = isglobal
+				isglobal = isglobal,
+				localentid = localentid
```

Comment:
I don't think we need to send both `isglobal` and `localentid`. We could send just an entity ID, and use worldspawn to represent global. (Then the `holoClip` functions that take `isglobal` would need to sent `this:EntIndex()` instead of `0` as the `localentid`.)

Reply:
So... literally exactly what I had in the last PR I made about this...